README.adoc

```diff
@@ -12,9 +12,9 @@ This repository contains the Trento project documentation that does not interest
 
 == Trento docs site
 
-The `trento-docs-site` directory builds the link:https://www.trento-project.io/docs/[Trento project’s unified documentation site] using https://antora.org/[Antora]. Learn more about xref:trento-docs-site/README.adoc[the documentation hub here].
+The `trento-docs-site` directory builds the link:https://www.trento-project.io/docs/[Trento project’s unified documentation site] using https://antora.org/[Antora]. Learn more about xref:content/trento-docs-site/README.adoc[the documentation hub here].
```

Comment:
Hey @nelsonkopliku , no need to point to content directory, in line [2](https://github.com/trento-project/docs/pull/108/files#diff-1194561cbea346acb1bea37d81fdf5c70def982dcbb0c64eca2a96bc51941124R2) we use a little trick to have navigation on antora page and github working. 

So the reader stays in the place where he is right now